core/backend/src/test/integration/HubUtility.ts

```diff
@@ -74,16 +63,20 @@ export class HubUtility {
   }
 
   /**
-   * Queries the project id by its name
+   * Queries the iTwin id by its name
    * @param requestContext The client request context
-   * @param projectName Name of project
-   * @throws If the project is not found, or there is more than one project with the supplied name
+   * @param name Name of iTwin
+   * @throws If the iTwin is not found, or there is more than one iTwin with the supplied name
    */
-  public static async queryProjectIdByName(requestContext: AuthorizedClientRequestContext, projectName: string): Promise<string> {
-    const project = await HubUtility.queryContextByName(requestContext, projectName);
-    if (!project)
-      throw new Error(`Project ${projectName} not found`);
-    return project;
+  public static async getITwinIdByName(requestContext: AuthorizedClientRequestContext, name: string): Promise<string> {

```

Comment:
As mentioned before, anything in the HubUtility that is related to Context/iTwin should be in the mock so that we can remove all this duplication. (Again for a future PR)